src/hooks/use-assistant.ts

```diff
@@ -35,7 +36,10 @@ export const useAssistant = ( selectedSiteId: string ) => {
 	const addMessage = useCallback(
 		( content: string, role: 'user' | 'assistant', chatId?: string ) => {
 			setMessages( ( prevMessages ) => {
-				const updatedMessages = [ ...prevMessages, { content, role, id: prevMessages.length } ];
+				const updatedMessages = [
+					...prevMessages,
+					{ content, role, id: prevMessages.length, creationDate: Date.now() },
```

Comment:
[Proposal/Suggestion] What about renaming `creationDate` to `createdAt`? I find it a bit redundant, and not 100% accurate ( as date.now, returns milliseconds, and not a date )

Reply:
To be honest, I wasn't super confident about the naming, `createdAt` sounds better to me. Thanks for the suggestion 🙇 !